imports/subscriptions/BoardsSubscriber.ts

```diff
@@ -100,4 +108,53 @@ class BoardsSubscriberClass {
   }
 }
 
+class BoardFavoritesSubscriberClass {
```

Comment:
Can we make this subscription into a seperate file? like `BoardFavoritesSubscriber.ts`